earn/src/components/lend/BorrowingWidget.tsx

```diff
@@ -0,0 +1,383 @@
+                      <Display size='XS' color={SECONDARY_COLOR}>
+                        {entry.asset.symbol}
+                      </Display>
+                    </div>
+                    <Display size='XXS' color={SECONDARY_COLOR}>
+                      {ltvText}
+                    </Display>
+                  </AvailableContainer>
+                );
+              })}
+            </div>
+          </div>
+        </CardContainer>
+      </CardWrapper>
+      <CardWrapper $textAlignment='end'>
+        <Text size='XL'>Borrows</Text>
+        <CardContainer>
+          <div>
+            <Text size='S' color={SECONDARY_COLOR}>
+              Active
+            </Text>
+            <div className='flex flex-col gap-2 mt-2'>
+              {marginAccounts &&
+                marginAccounts.map((account) => {
+                  const hasAssetsForToken0 = account.assets.token0Raw > 0;
+                  const hasAssetsForToken1 = account.assets.token1Raw > 0;
+                  const hasLiabilitiesForToken0 = account.liabilities.amount0 > 0;
+                  const hasLiabilitiesForToken1 = account.liabilities.amount1 > 0;
+                  const token0Color = tokenColors.get(account.token0.address);
+                  const token0Gradient = token0Color
+                    ? `linear-gradient(90deg, ${GREY_700} 0%, ${rgba(token0Color, 0.25)} 100%)`
+                    : undefined;
+                  const token1Color = tokenColors.get(account.token1.address);
+                  const token1Gradient = token1Color
+                    ? `linear-gradient(90deg, ${GREY_700}  0%, ${rgba(token1Color, 0.25)} 100%)`
+                    : undefined;
+                  return (
+                    <Fragment key={account.uniswapPool}>
+                      {hasLiabilitiesForToken0 && (
+                        <AvailableContainerConnectedRight $backgroundGradient={token0Gradient}>
+                          <Display size='XXS' color={SECONDARY_COLOR}>
+                            3% APY
+                          </Display>
+                          <div className='flex items-end gap-1'>
+                            <Display size='S' color={SECONDARY_COLOR}>
+                              {formatTokenAmount(account.liabilities.amount0)}
+                            </Display>
+                            <Display size='XS' color={SECONDARY_COLOR}>
+                              {account.token0.symbol}
+                            </Display>
+                          </div>
+                        </AvailableContainerConnectedRight>
+                      )}
+                      {hasAssetsForToken0 && !hasLiabilitiesForToken1 && <AvailableContainerPlaceholder />}
+                      {hasLiabilitiesForToken1 && (
+                        <AvailableContainerConnectedRight $backgroundGradient={token1Gradient}>
+                          <Display size='XXS' color={SECONDARY_COLOR}>
+                            3% APY
```

Comment:
?

Reply:
Didn't get around to computing the actual value here